### ISSUE TITLE
app.route() code is error ? [guide/routing.md]

### ISSUE THREAD

**Issue by 0neSe7en** [expressjs/expressjs.com#369]
this code

``` javascript
app.route('/book')
  .get(function(req, res) {
    res.send('Get a random book');
  });
  .post(function(req, res) {
    res.send('Add a book');
  });
  .put(function(req, res) {
    res.send('Update the book');
  });
```

should be

``` javascript
app.route('/book')
    .get(function(req, res) {
      res.send('Get a random book');
    })
    .post(function(req, res) {
      res.send('Add a book');
    })
    .put(function(req, res) {
      res.send('Update the book');
    })
;
```

sorry for my poor English.

**Comment by dougwilson** [comment]
Yes, those semicolons should not be there. It's an error from PR #362 @hacksparrow can you audit all the semicolons that were added in that PR? it looks like it may have just been a find-replace job instead of an actual "add where they need to be".

**Comment by dougwilson** [comment]
Fixed by #370
